### ISSUE TITLE
Display Server status & connectivity on the web site

### ISSUE THREAD

**Issue by k33k00** [AHAAAAAAA/PokemonGo-Map#1171]
## Expected Behavior
When the servers are down it would be nice to have something on the site stating what is going on. 

## Current Behavior
Absolutely nothing, eventually you're left with a blank map and no idea what is going on. Unless you have debug mode and the console in front of you.

## Possible Solution
Add a toaster style popup to the site stating the current issue. 
# For example: 
- When the user can't login to the go server have a pop up saying - PokemonGo login server is down \<timestamp\>.  
- When a request for data fails - have a pop up saying - request for \<data\> failed at \<timestamp\> [\<httpresponse\> | "Pokemon servers down"]

![sign in failure](https://cloud.githubusercontent.com/assets/3812864/17052425/50854990-4ff5-11e6-9f29-a00786c52ef9.jpg)

## Steps to Reproduce (for bugs)
1. Just watch it for a few hours, the servers will go down and you'll see it happen

## Context
I was running the server, and have been successfully for about 2 days now, constantly updating to the latest commits. When something goes wrong I never know whether it's the latest version or the servers. Obviously running debug mode while I'm sitting watching is helpful, though I can't be watching all the time. Any way I end up with a blank map if the servers are down for long enough (~15 minutes) with no reason as to why

## Your Environment
* Version used: develop branch
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Ubuntu server 14.04 x64

**Comment by Valareos** [comment]
I agree this would be a nice feature. at the moment I use http://ispokemongodownornot.com/ to determine if its the servers or me

**Comment by FrostTheFox** [comment]
It would be nice to implement, but for right now ispokemongodownornot.com or go.jooas.com work pretty well.

**Comment by k33k00** [comment]
I've found what looks to be a [nice toaster popup library](http://codeseven.github.io/toastr/demo.html). I'm guessing the hard part is the getting python and JS to interact

**Comment by MetalH47K** [comment]
Maybe at the start of each scan it can try to log in again with the account details to check if the server is still up. So it would be once every couple of minutes.
